### PR TITLE
Fix checks with comma issues in checks @j2clerck

### PR DESCRIPTION
**Pull request by j2clerck**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checks/check122 [hunk 1]
```diff
@@ -42,9 +42,9 @@ check122(){
         textFail "$REGION: Policy $policy allows \"*:*\"" "$REGION" "$policy"
       done
     else
-        textPass "$REGION: No custom policy found that allow full \"*:*\" administrative privileges" "$REGION" "$policy"
+        textPass "$REGION: No custom policy found that allow full \"*:*\" administrative privileges" "$REGION"
```
Comment (toniblyx) [inline checks/check122:45]: Why do you remove policy here and you don´t in textFail? it would brake the csv anyway.

File: checks/check_extra7149 [hunk 2]
```diff
@@ -31,7 +31,7 @@ extra7149() {
             for redshiftcluster in $LIST_OF_REDSHIFT_CLUSTERS; do
                 REDSHIFT_SNAPSHOT_ENABLED=$($AWSCLI redshift describe-cluster-snapshots $PROFILE_OPT --region $regx --cluster-identifier $redshiftcluster --snapshot-type automated)
                 if [[ $REDSHIFT_SNAPSHOT_ENABLED ]]; then
-                    textPass "$regx: Redshift cluster $redshiftcluster has automated snapshots $REDSHIFT_SNAPSHOT_ENABLED" "$regx" "$redshiftcluster"
+                    textPass "$regx: Redshift cluster $redshiftcluster has automated snapshots." "$regx" "$redshiftcluster"
```
Comment (toniblyx) [inline checks/check_extra7149:34]: why this change?

**Review by toniblyx — CHANGES_REQUESTED** [review]

File: checks/check122 (hunk 1 above)
Comment (toniblyx) [reply checks/check122:45]: Reading the discord chat, I understand that point but why not to sanitize the variable `$policy` to make sure it doesn't  include a comma? 

Something like `echo $policy | awk -F',' '{ print $1 }'`

Comment (toniblyx) [reply checks/check122:45]: having resource name in any status helps or will help for visualizations

**Comment by toniblyx** [comment]
As mentioned in Discord, running `grep \, checks/*| grep -E "textFail|textPass|textInfo|RISK|REMEDIATION"` I get extra7152, extra7153, extra7156, extra7162, extra765, extra769 and extra794 that have to be changed. Let me know if you can make it or I can do it.

File: checks/check122 (hunk 1 above)
Comment (j2clerck) [reply checks/check122:45]: If you look at the for loop, you will issue a textPass once to tell that no policy contains '*:*', so it is not relevant to add the latest policy checked, or you should have one textPass per policy and include the PolicyName.
Regarding the textFail, it triggers for each policy that fail and the variable policy is sanitized in this case.

File: checks/check_extra7149 (hunk 2 above)
Comment (j2clerck) [reply checks/check_extra7149:34]: REDSHIFT_SNAPSHOT_ENABLED is a json output. It does not format well, contains commas and multi-line.

**Comment by j2clerck** [comment]
@toniblyx I will have a look at the remaining checks, but probably later in the day (Eastern Time)

**Comment by toniblyx** [comment]
Awesome, thanks @j2clerck

File: checks/check_extra726 [hunk 3]
```diff
@@ -47,7 +47,7 @@ extra726(){
         textInfo "$REGION: Trusted Advisor check $TA_CHECKS_NAME is in warning state $QUERY_TA_CHECK_RESULT" "$REGION" "$TA_CHECKS_NAME"
         ;;
       "not_available")
-        textInfo "$REGION: Trusted Advisor check $TA_CHECKS_NAME is in not_available state $QUERY_TA_CHECK_RESULT" "u$REGION" "$TA_CHECKS_NAME"
+        textInfo "$REGION: Trusted Advisor check $TA_CHECKS_NAME is in not_available state $QUERY_TA_CHECK_RESULT" "$REGION" "$TA_CHECKS_NAME"
```
Comment (j2clerck) [inline checks/check_extra726:50]: There was a typo in the region name which was prefixed by u

File: checks/check122 (hunk 1 above)
Comment (toniblyx) [reply checks/check122:45]: got it

File: checks/check_extra726 (hunk 3 above)
Comment (toniblyx) [reply checks/check_extra726:50]: thanks

**Review by toniblyx — APPROVED** [review]
